examples/partial.rs

```diff
@@ -16,23 +15,25 @@ fn main_widget() -> impl Widget<TestData> {
         .with_child(Radio::new("Variant B", TestData::B(String::from("hi"))))
         .with_child(Radio::new("Variant C", TestData::C));
 
-    // let partial = PrismWrap::with_closures(
-    //     TextBox::new(),
-    //     String::new(),
-    //     |outer: &TestData| {
-    //         if let TestData::B(str) = outer {
-    //             Some(str.clone())
-    //         } else {
-    //             None
-    //         }
-    //     },
-    //     TestData::B,
-    // );
+    let partial = DisablePrismWrap::new(
+        TextBox::new(),
+        String::new(),
+        Closures(
+            |outer: &TestData| {
+                if let TestData::B(str) = outer {
```

Comment:
Minor suggestion: avoid using `str` as variable name since it's also the name of a type (string slice `&str`).

Reply:
Ok, i changed it